src/frontend/src/helpers/Services.ts

```diff
@@ -106,12 +106,20 @@ namespace Services {
 
     // login successful
     await localStorage.connect(`seashell-${connection.username}`);
+    await localStorage.deleteDB();
+    await localStorage.connect(`seashell-${connection.username}`);
```

Comment:
This is just for development environment. Either you clear the DB manually, or I clear it here. For production I will remove it from here.